addons/winddeflection/stringtable.xml

```diff
@@ -11,16 +11,18 @@
                 <French>Vent</French>
                 <German>Windinformationen</German>
                 <Hungarian>Szélinformáció</Hungarian>
+                <Italian>Informazioni sul vento</Italian>
             </Key>
             <Key ID="STR_ACE_WEATHER_METER_WIND_DIRECTION">
                 <English>Direction: %1</English>
                 <Polish>Kierunek: %1</Polish>
                 <Spanish>Dirección: %1</Spanish>
-                <Russian>Направление: %1</Russian>
+                <Russian>Направление: %1°</Russian>
```

Comment:
As you wish, as long as the angular degrees are meant to be inside
placeholder, nor MOA's or something.